### DISCUSSION TITLE
Provide username and password for website login

### DISCUSSION THREAD

**Discussion by andy1463** [1Remote/1Remote#814] [Q&A]
Hello,
is it possible to deposit username and password for a website that I don't need to type it every time if I start it via app tab?
Thanks a lot and best regards
Andreas

**Comment by majkinetor** [comment]
Its possible in general. 

Can you provide more info?

**Reply by andy1463** [reply]
I started a browser in app tab.
In CMD parameter I added the webaddress under prefix.
Everything is working. Now I want to save username and password, that I don't need to type it everytime but have no idea how to get this working.

**Reply by majkinetor** [reply]
That's not supported out of the box.

[Browser can't make it work too](https://stackoverflow.com/questions/7103996/javascript-bookmarklet-go-to-url-and-execute). It can be done via [AHK](https://www.autohotkey.com/) in 3 lines of code. Start the browser, wait to load, send keys as a bookmarklet (the same as on SO, just not via browser).

**Reply by majkinetor** [reply]
Perhaps it could be done as 1Remote feature, but it would need to do equivalent of AHK mentioned above.

**Reply by majkinetor** [reply]
Why don't use native browser password saving and reapplying features. I recognize it's not ideal, but it would work.

**Reply by VShawn** [reply]
looks overkill for 1Remote. 1password / lastpass / bitwarden has already done well in this field.
